src/main/java/org/kohsuke/github/GHAppCreateTokenBuilder.java

```diff
@@ -46,6 +46,20 @@ public GHAppCreateTokenBuilder repositoryIds(List<Long> repositoryIds) {
         return this;
     }
 
+    /**
+     * By default the installation token has access to all repositories that the installation can access. To restrict
+     * the access to specific repositories, you can provide repository names when creating the token.
+     *
+     * @param repositories
+     *            Array containing the repository names
+     * @return a GHAppCreateTokenBuilder
+     */
+    @BetaApi
```

Comment:
From a bit of reading I think this API is out of beta now, but I added this annotation anyways to match the other methods in this class. Let me know if I should remove the annotation or anything.